models/klaviyo__person_campaign_flow.sql

```diff
@@ -0,0 +1,35 @@
+with events as (
+    select *
+    from {{ ref('klaviyo__events') }}
+),
+
+pivot_out_events as (
+    
+    select 
+        person_id,
+        last_touch_campaign_id,
+        last_touch_flow_id,
+        campaign_name,
+        flow_name,
+        variation_id,
+        min(occurred_at) as first_event_at,
+        max(occurred_at) as last_event_at
+
+    -- sum up the numeric value associated with events (most likely will mean revenue)
+    {% for rm in var('klaviyo__sum_revenue_metrics') %}
+    , sum(case when lower(type) = '{{ rm | lower }}' then numeric_value else 0 end) 
+        as {{ 'sum_revenue_' ~ rm | replace(' ', '_') | replace('(', '') | replace(')', '') | lower }} -- removing special characters that I have seen in different integration events
```

Comment:
Do you know if there are possibly other replace functions we would need to include in order to be able to capture other character that are in these fields? 

dbt 0.19.0 introduced the ability to use the re python package which could be used to dynamically replace characters.

Reply:
While I think the above would be great, that would require all users of this package to be on dbt 19 or greater. This can be something we re-address later if needed.

Reply:
yeah the only special characters i saw in the sample data event types were parentheses and spaces. if some other ones come up with customers, it could definitely be worth creating some sort of dynamic-replacement macro that isn't so strict regarding the dbt version (plus it seems like we might need to look into that for jira anyways)